r2dbc/boot-r2dbc-json-column/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM eclipse-temurin:21.0.5_11-jre-alpine as builder
+WORKDIR ./app
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)